.github/ISSUE_TEMPLATE/release.yaml

```
@@ -0,0 +1,100 @@
+name:  Release
+description: Release/Pre-release of a version.
+labels: [release]
+body:
+- type: dropdown
+  id: release_type
+  attributes:
+    label: Release Type
+    description: which type of release is this release?
+    options:
+    - major release
+    - minor release
+    - patch release
+- type: textarea
+  id: release_version
+  attributes:
+    label: Release Version
+    description: Version of this release (e.g `2.4.0`)
+    placeholder:
+- type: checkboxes
+  id: release_tests
+  attributes:
+    label: "**For all releases** Github Workflow Test Matrix Checkup"
+    description: |
+      We maintain some integration tests with 3rd party components which we need to manually verify and update before cutting any release.
+      An issue exists to automate the following actions: https://github.com/Kong/kubernetes-ingress-controller/issues/1886
+      **Wait for tests to pass before continuing with any release**, if any problems are found hold on the release until patches are provided and then run the tests again.
+    options:
+      - label: check the testing workflow ([.github/workflows/test.yaml](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/test.yaml)) and ensure that all matrix versions ([`.github/workflows/e2e.yaml`](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/e2e.yaml) and [`.github/workflows/release.yaml`](/Kong/kubernetes-ingress-controller/blob/main/.github/workflows/release.yaml))  are up to date for various component releases. If there have been any new releases (major, minor or patch) of those components since the latest version seen in that configuration make sure the new versions get added before proceeding with the release. Remove any versions that are no longer supported by the environment provider.
+        required: true
+      - label: Kubernetes
+        required: true
+      - label: Istio
+        required: true
+- type: checkboxes
+  id: release_branch
+  attributes:
+    label: "**For all releases** Create Release Branch"
+    description: For this step we're going to start with the `main` branch to create our release branch (e.g. `release/X.Y.Z`) which will later be submitted as a pull request back to `main`.
+    options: 
+      - label: "ensure that you have up to date copy of `main`: git fetch --all"
+        required: true
+      - label: "create the release branch for the version (e.g. `release/1.3.1`): `git branch -m release/x.y.z`"
+        required: true
```

Folding the description into a step, minor formatting fix, and correcting an inaccuracy in the original (we want a new branch, not to rename the existing one).

```suggestion
    options: 
      - label: "ensure that you have up to date copy of `main`: `git fetch --all`"
        required: true
      - label: "check out `main`: `git checkout main`"
        required: true
      - label: "create the release branch for the version (e.g. `release/1.3.1`): `git checkout -b release/x.y.z`"
        required: true
```

---

removed `description` part